package.json

```diff
@@ -20,7 +20,7 @@
     "gatsby-plugin-manifest": "^2.4.35",
     "gatsby-plugin-mdx": "^1.2.47",
     "gatsby-plugin-meta-redirect": "^1.1.1",
-    "gatsby-plugin-newrelic": "newrelic/gatsby-plugin-newrelic.git#5cd42e832f0588764dc332ce7ebd05794de5106a",
+    "gatsby-plugin-newrelic": "^1.0.5",
```

Comment:
Since this is installed from the theme, do we need this here?

Reply:
I guess not? I'll remove it